updating enterprise documentation with additional clarity (#7495)
hashicorp/consul: website/source/docs/enterprise/backups/index.html.md

page_title: "Consul Enterprise Automated Backups"
sidebar_current: "docs-enterprise-backups"
description: |-
  Consul Enterprise provides a highly available service that manages taking snapshots, rotation and sending backup files offsite to Amazon S3 (or S3-compatible endpoints), Google Cloud Storage, or Azure Blob Storage.
---

# Automated Backups

> [Consul Enterprise](https://www.hashicorp.com/consul.html) enables customers to run

Reviewer: ```suggestion
[Consul Enterprise](https://www.hashicorp.com/consul.html) enables you to run
```

> the snapshot agent within their environment as a service (Systemd as an example)

Reviewer: ```suggestion
the snapshot agent within your environment as a service (Systemd as an example)
```

> or scheduled through other means. Once running, this service operates as a highly 

Reviewer: ```suggestion
or scheduled through other means. Once running, the snapshot agent service operates as a highly 
```

available process that integrates with the snapshot API to automatically manage 
taking snapshots, backup rotation, and sending backup files offsite to Amazon S3 
(or another S3-compatible endpoint), Google Cloud Storage, or Azure Blob Storage.

This capability provides an enterprise solution for backup and restoring the state of Consul servers 

> within an environment in an automated manner. These Snapshots are atomic and point-in-time. Consul 

Reviewer: ```suggestion
within an environment in an automated manner. These snapshots are atomic and point-in-time. Consul 
```

> cluster backups include: 

Reviewer: ```suggestion
datacenter backups include: 
```

Reviewer: I'd add language around "include but is not limited to"

- Key/Value Store Entries
- Service Catalog Registrations 
- Prepared Queries 
- Sessions
- Access Control Lists (ACLs)
- Namespaces

> For more experience leveraging Consuls snapshot functionality, we suggest you look through our HashiCorp 

Reviewer: ```suggestion
For more experience leveraging Consul's snapshot functionality, we suggest you look through our HashiCorp 
```

Learn guide for [Datacenter Backups in Consul](https://learn.hashicorp.com/consul/datacenter-deploy/backup). 

> For detailed configuration information on configuring this functional, review the 

Reviewer: ```suggestion
For detailed configuration information on configuring the Consul Enterprise's snapshot agent, review the 
```
